doc/source/whatsnew/v0.20.0.txt

```diff
@@ -499,7 +499,7 @@ Other Enhancements
 - ``DataFrame.to_excel()`` has a new ``freeze_panes`` parameter to turn on Freeze Panes when exporting to Excel (:issue:`15160`)
 - ``pd.read_html()`` will parse multiple header rows, creating a MutliIndex header. (:issue:`13434`).
 - HTML table output skips ``colspan`` or ``rowspan`` attribute if equal to 1. (:issue:`15403`)
-- :class:`pandas.io.formats.style.Styler` template now has blocks for easier extension, :ref:`see the example notebook <style.ipynb#Subclassing>` (:issue:`15649`)
```

Comment:
We can also pin nbsphinx to 0.3.1 (the new release breaking this is only from last day) 

(but not sure if that is worth it for this one link ..)

Reply:
If you use `:ref:`-style links, the link has to be relative to the base source directory and it has to start with as slash (since version 0.3.0, see https://github.com/spatialaudio/nbsphinx/blob/master/NEWS.rst and the link mentioned above, look at the "Note" section in the end).
This is a breaking change between 0.3.0 and older versions.

AFAICT, the issue https://github.com/spatialaudio/nbsphinx/issues/174 is *not* related to `:ref:`-style links. It's a problem with re-writing "normal" reST links and its interactions with the "plot directive" from matplotlib (but it might also appear with other extensions?).

Reply:
@mgeier Thanks for the input! So it should just be the slash that is missing.

Reply:
Thanks @mgeier, somehow I read that note about the `/` multiple times and missed it :)